contact.php

```diff
@@ -1,20 +1,29 @@
 <?php require_once __DIR__ .'/variables.php'?>
 <!DOCTYPE html>
 <html lang="en">
-<?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?=$titleContact?></title>
+<?php include_once './template/head.phtml';?>
+<?php
+function getPageTitle($page_titles) {
+    $title = $page_titles[basename($_SERVER['SCRIPT_FILENAME'])];
+    return $title;
+}
+?>
+<title><?php echo $title = getPageTitle($page_titles); ?></title>
 <body>
    <header>
+       <nav>
        <?php require_once __DIR__ .'/template/menu.phtml';?>
-    <ul class="menu-nav">
-        <?php
-        foreach ($menuitems as $url => $label){
-        ?>
-        <li class="nav-item ">
-            <a href="<?=  $url ?>" class="nav-link"><?=$label?></a>
-            <?php } ?>
-        </li>
-    </ul>
+       <ul class="menu-nav">
+           <?php
+           $currentPage = basename($_SERVER['SCRIPT_FILENAME']);

```

Comment:
` $currentPage = basename($_SERVER['SCRIPT_FILENAME']);`
Can it defined globally instead of every page?

Reply:
done)